packages/shared/src/components/post/ShareYouTubeContent.tsx

```diff
@@ -1,18 +1,31 @@
-import React, { ReactElement } from 'react';
+import React, { ReactElement, useEffect } from 'react';
+import { useMutation } from '@tanstack/react-query';
 import PostSourceInfo from './PostSourceInfo';
-import { Post } from '../../graphql/posts';
+import { Post, sendViewPost } from '../../graphql/posts';
 import { SharePostTitle } from './share';
 import { SharedLinkContainer } from './common/SharedLinkContainer';
 import { SharedPostLink } from './common/SharedPostLink';
 import YoutubeVideo from '../video/YoutubeVideo';
 import { formatReadTime } from '../utilities';
+import { useAuthContext } from '../../contexts/AuthContext';
 
 interface ShareYouTubeContentProps {
   post: Post;
   onReadArticle: () => Promise<void>;
 }
 
 function ShareYouTubeContent({ post }: ShareYouTubeContentProps): ReactElement {
```

Comment:
This is the one for the share modal, I think they already fire it, but not 100% sure for youtube ones.
So maybe double check we don't fire it twice there now?

Reply:
Ah, right. It's already in `SquadPostContent`